Source/FluentSpreadsheets.GoogleSheets/Extensions/IndexRangeIndex.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Google.Apis.Sheets.v4.Data;
+
+namespace FluentSpreadsheets.GoogleSheets.Extensions;
+
+internal static class IndexRangeExtensions
+{
+    public static GridRange ToGridRange(this IndexRange range, int sheetId)
+    {
+        return new GridRange
+        {
+            StartRowIndex = range.Start.Row - 1,
+            StartColumnIndex = range.Start.Column - 1,
+            EndRowIndex = range.End.Row - 1,
+            EndColumnIndex = range.End.Column - 1,
+            SheetId = sheetId
+        };
```

Comment:
if google sheets have a zero-based indices, you should just start renderer at (0, 0), not decrement it in the range converter.

Reply:
It's a bit unclear, we are using 2 kinds of indices, in this file they're starting from 0, but when it's in format "sheetName!A1" we start from 1. I think we should leave it the way it is.

Reply:
ok